packages/trader/src/Modules/Trading/Containers/Multiplier/multiplier-amount-modal.tsx

```diff
@@ -76,7 +78,7 @@ const TradeParamsMobile = observer(({ toggleModal }) => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [stake_value]);
 
-    const setSelectedAmount = (basis, stake) => {
```

Comment:
Parameter `basis` wasn't used inside of the function, so I deleted it here and in places where that function was called.